packages/unmock-core/src/__tests__/composition.test.ts

```diff
@@ -0,0 +1,57 @@
+import {
```

Comment:
Check out https://jestjs.io/docs/en/es6-class-mocks - it is a cleaner way to mock classes.

Reply:
Not sure how these would work w/o a class; can't find a way to use an interface cleanly with `jest.mock` in a way that TS won't complain.

Reply:
Or did you mean just using `jest.fn()` in all pseudo-functions?